nautobot/extras/templates/extras/dynamicgroup_edit.html

```diff
@@ -62,13 +62,15 @@
                                 {% if forloop.first %}
                                     <thead>
                                         <tr>
+                                            <th>Sort</th>
```

Comment:
I'm thinking we might still want to call this column **Weight** or something that is indicative that ordering is significant?